pkgs/by-name/py/pyfa/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "pyfa";
+  version = "2.60.0";
+  src = fetchurl {
+    url = "https://github.com/pyfa-org/Pyfa/releases/download/v${version}/pyfa-v${version}-linux.AppImage";
+    name = "pyfa-${version}";
+    sha256 = "sha256-bPInlYtsHtLk3AkF8Bf9CgJUAX7zRvdb6qO11MzF3eE=";
+  };
+  appimageContents = appimageTools.extractType1 { inherit pname version src; };
+in
+appimageTools.wrapType1 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    install -m 444 -D ${appimageContents}/pyfa.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/pyfa.desktop \
+      --replace-warn 'Exec=pyfa' 'Exec=${pname}'
```

Comment:
Is this relevant?

Reply:
Isn't it by default a noop? I think we should dropt this.

Reply:
What is a noop? 

I guess I can remove this, I used make desktop item in the flake but wasn't sure how to use this as package